modules/caddyhttp/templates/tplcontext.go

```diff
@@ -79,6 +79,7 @@ func (c *TemplateContext) NewTemplate(tplName string) *template.Template {
 		"markdown":         c.funcMarkdown,
 		"splitFrontMatter": c.funcSplitFrontMatter,
 		"listFiles":        c.funcListFiles,
+		"isDir":            c.funcIsDir,
```

Comment:
```suggestion
		"stat":            c.funcIsDir,
```

(Maybe rename `funcIsDir` to `funcStat`)